applications/maplab-server-node/cfg/maplab_server_ros_params.yaml

```diff
@@ -6,7 +6,7 @@ num_hardware_threads:                                                 12 # Set i
 
 # INPUT / OUTPUT
 ############################
-maplab_server_backup_interval_s:                                      300
+maplab_server_backup_interval_s:                                      10
```

Comment:
This is a separate thread, right (Haven't checked the code below yet)?  Are`10s` affecting our performance? Sounds quite often.

Reply:
actually this yaml was included in the commit only by accident and is not used by our pipeline
anyway this does not actually save the map every `10s`. what actually happens is that after every optimization cycle we check if the last time the map was saved was more than `10s` ago and if this is the case it saves the map. So I just set it so low to make sure the map is saved after every full optimization cycle